fleet-unit-healthcheck.go

```diff
@@ -92,13 +117,59 @@ func fleetUnitHealthHandler(fleetAPIClient client.API, checker fleetUnitHealthCh
 }
 
 func newFleetUnitHealthCheck(unitState schema.UnitState, checker fleetUnitHealthChecker) fthealth.Check {
+	name := unitState.Name
+	if strings.Contains(name, "sidekick") {
+		return buildHealthcheck(unitState, checker, 3, sidekickTecSum, sidekickBusImp)
+	} else if strings.Contains(name, "bridge") {
+		return buildHealthcheck(unitState, checker, 3, bridgeTecSum, bridgeBusImp)
+	} else if strings.Contains(name, "kafka") || strings.Contains(name, "zookeeper") {
+		return buildHealthcheck(unitState, checker, 1, queueTecSum, queueBusImp)
+	} else if strings.Contains(name, "varnish") {
+		return buildHealthcheck(unitState, checker, 1, varnishTecSum, varnishBusSum)
+	} else if strings.Contains(name, "vulcan") {
+		return buildHealthcheck(unitState, checker, 1, vulcanTecSum, vulcanBusSum)
+	} else if strings.Contains(name, "aggregate-healthcheck") {
+		return buildHealthcheck(unitState, checker, 1, aggHcTecSum, aggHcBusSum)
+	} else if strings.Contains(name, "timer") {
```

Comment:
Is it a fair assumption that all our timers will always be database backup related?